### PR TITLE
Updates NPM to Node.js

### LINKED ISSUE #802: Update nodejs installation
brew isnt the best experience - just use nodejs.org .pkg file.

### PR DESCRIPTION
**Pull request by diana-lakatos**
Fixes #802

### CODE REVIEW

File: app/views/pages/get-started/installing-nodejs-mac.liquid [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Installing Node.js on a Mac is a two-step process:
+
+<div data-autosteps></div>
+
+### Step 1: Install Node.js
+
+Go to [nodejs.org](https://nodejs.org/en/). When visiting the page from a Mac, you will see download links for MacOS. Select the LTS (Long Term Support) version. The Current version has the latest features, but may be more prone to changes and bugs than the LTS version.
+
+When the download has finished, locate the doanloaded file in Finder and double-click on it. Go through the installation steps as instructed by the installer.
```
Comment (pavelloz) [inline app/views/pages/get-started/installing-nodejs-mac.liquid]:
```suggestion
When the download has finished, locate the downloaded file in Finder and double-click on it. Go through the installation steps as instructed by the installer.
```

File: app/views/pages/get-started/installing-nodejs-mac.liquid [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+### Step 2: Check if Node.js is installed
+
+To check if Node.js and NPM are installed, enter:
+
+* `node -v` in Terminal to see the version of Node.js installed
+* `npm -v` in Terminal to see the version of NPM installed
+
+The version number of Node.js/NPM will be displayed.
+
+To upgrade to the latest version of Node.js, enter `brew update` (to update the version of Node.js in Homebrew), and `brew upgrade node`(to upgrade Node.js).
```
Comment (pavelloz) [inline app/views/pages/get-started/installing-nodejs-mac.liquid]:
```suggestion
To upgrade to the latest version of Node.js, enter `brew update` (to update the version of Node.js in Homebrew), and `brew upgrade node`(to upgrade Node.js).
```

This is not applicable anymore. Upgrading is doing the same as installation, just with new installation file from nodejs.org

File: app/views/pages/get-started/installing-nodejs-mac.liquid (hunk 1 above)
Comment (diana-lakatos) [reply app/views/pages/get-started/installing-nodejs-mac.liquid]: Thank you!

File: app/views/pages/get-started/installing-nodejs-mac.liquid (hunk 2 above)
Comment (diana-lakatos) [reply app/views/pages/get-started/installing-nodejs-mac.liquid]: Thank you! This is a leftover from the brew version, I will remove it.

**Review by pavelloz — APPROVED** [review]

**Pull request merged**